builds/e2e/templates/deploy-iotuap-setup.yaml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  edgelet.artifact.name: ''
+  images.artifact.name: ''
+
+steps:
+  - task: CopyFiles@2
+    displayName: 'Copy Edgelet Artifact'
+    inputs:
+      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['edgelet.artifact.name'] }}"
+      TargetFolder: "\\\\$(iotuap-x64-machine)\\c$\\Data\\e2etests\\artifacts\\${{ parameters['edgelet.artifact.name'] }}"
```

Comment:
do you need to create e2etests folder if it doesn't exist?

Reply:
I do not think this is needed. This is how it's been done for the other Windows tests. I have not seen any issues there. Perhaps the _CopyFiles_ task handles that differently than the _PowerShell_ task that would fail.